### PR TITLE
Fix remaining lint issues

### PR DESCRIPTION
**Pull request by Frassle**
All paralleltest issues, also turns on the linter for these files now they lint ok.

### CODE REVIEW

File: tests/integration/integration_python_acceptance_test.go [hunk 1]
```diff
@@ -238,8 +247,6 @@ func TestAutomaticVenvCreation(t *testing.T) {
 	// handling by the pulumi CLI itself.
 
 	check := func(t *testing.T, venvPathTemplate string, dir string) {
-		t.Parallel()
-
```
Comment (dixler) [inline tests/integration/integration_python_acceptance_test.go:242]: Why was this removed?

File: .golangci.yml [hunk 2]
```diff
@@ -9,6 +9,8 @@ run:
     - third_party$
     - vendor$
   skip-dirs-use-default: false
+  build-tags:
+  - all
```
Comment (dixler) [inline .golangci.yml:13]: What's this for?

**Review by dixler — COMMENTED** [review]
just have 2 questions

File: .golangci.yml (hunk 2 above)
Comment (Frassle) [reply .golangci.yml:13]: A load of the test files have build tags on them so they only build (or lint) if that tag is set. e.g. `tests/integration/aliases/aliases_go_test.go` has `//go:build (go || all) && !xplatform_acceptance` at the top of it.

`all` is just the tag we use to save us having to write `python,nodejs,go` longhand to compile these files.

File: tests/integration/integration_python_acceptance_test.go (hunk 1 above)
Comment (Frassle) [reply tests/integration/integration_python_acceptance_test.go:242]: I moved it out of the lambda and into the call sites, eg see:
```
t.Run("RelativePath", func(t *testing.T) {
		t.Parallel()
		check(t, "venv", filepath.Join("python", "venv"))
	})
```

The linter understands this, so it was either do this, or comment all the t.Runs below with `//nolint:paralleltest // check calls t.Parallel()`. I felt this style was nicer.

**Review by dixler — APPROVED** [review]
